src/colourConverters.ts

```diff
@@ -0,0 +1,37 @@
+import type { ColourName } from './types/namedColours';
+import { NamedColours } from './types/namedColours';
+
+export const nameToHex = (name: ColourName): string => {
+	const hexcode = NamedColours[name];
+	return hexcode;
+};
+
+export const hexToRGB = (hex: string): number[] | null => {
+	const shorthandRegex = /^#?([a-f\d])([a-f\d])([a-f\d])$/i;
```

Comment:
Do you want to enforce the preceding `#` everywhere?
```suggestion
	const shorthandRegex = /^#([a-f\d])([a-f\d])([a-f\d])$/i;
```